File: C++/Daily Flash/week1/Day4_Jan16/code_C++/prog2.cpp
/*
 * Program 2: Write a Program to calculate Simple Interest. Taking all essential
terms to compute as input.
*
*/

#include <iostream>

int main() {
	//take primary ammount and year
	int priAmt,year;

	//take interest rate
	float rate;


	std::cout <<"Enter the primary ammount: " << std::endl;
	std::cin >> priAmt;

	std::cout << "Enter the interest rate(peercentage): " <<std::endl;
	std::cin >> rate;

	std::cout << "Enter the year: " << std::endl;
	std::cin >> year;

	float simpleInterest = priAmt*year* (rate / 100);
	std::cout << "Simple interest = "<< simpleInterest << std::endl;
	return 0;
}

File: JAVA/Daily Flash Codes/Week1/Day4_Jan16/20Aug/prog2.java
import java.io.*;

class Program {

	public static void main(String args[]) throws IOException{
		
		int priAmt,year;
		//take interest rate
		int rate;
		double simpleInterest;

		BufferedReader br= new BufferedReader (new InputStreamReader (System.in));

		//priAmt
		System.out.println("Enter Primary ammount: ");
		priAmt = Integer.parseInt(br.readLine());

		//rate
		System.out.println("Enter interest Rate: ");
		rate = Integer.parseInt(br.readLine());

		//year
		System.out.println("Enter interest Rate: ");
		year = Integer.parseInt(br.readLine());
			
		//System.out.println(priAmt + " ");
		//System.out.println(rate + " ");
		//System.out.println(year + " ");

		simpleInterest = priAmt * year * (rate / 100.0);

		System.out.println("Simple interest ="+ simpleInterest);

		
	
	
	}

}	
